presto-main/src/test/java/com/facebook/presto/sql/planner/optimizations/TestMergeWindows.java

```diff
@@ -464,19 +468,21 @@ public void testNotMergeDifferentOrdering()
                                         .addFunction(functionCall("sum", COMMON_FRAME, ImmutableList.of(QUANTITY_ALIAS))),
                                 window(windowMatcherBuilder -> windowMatcherBuilder
                                                 .specification(specificationA)
-                                                .addFunction(functionCall("sum", COMMON_FRAME, ImmutableList.of(EXTENDEDPRICE_ALIAS)))
-                                                .addFunction(functionCall("sum", COMMON_FRAME, ImmutableList.of(DISCOUNT_ALIAS))),
+                                                .addFunction(functionCall("sum", UNSPECIFIED_FRAME, ImmutableList.of(EXTENDEDPRICE_ALIAS)))
```

Comment:
@ebd2 Thanks for the detailed explanation! I finally added the frame information to the plan that we verify in the test. Please take one more look.